### PR TITLE
Fixes for GA

### PR DESCRIPTION
**Pull request by cparisineti**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/CreateWorkerOptions.java [hunk 1]
```diff
@@ -128,8 +128,8 @@ public String getWorkerId() {
      * Returns queue assignments.
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/CreateWorkerOptions.java:128]: Update comment

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/DistributionPolicy.java [hunk 2]
```diff
@@ -77,26 +77,24 @@ public DistributionPolicy setName(String name) {
...
-    @Generated
-    public Double getOfferExpiresAfterSeconds() {
-        return this.offerExpiresAfterSeconds;
+    public Duration getOfferExpiresAfter() {
+        return this.offerExpiresAfter;
     }
 
     /**
-     * Set the offerExpiresAfterSeconds property: The number of seconds after which any offers created under this policy
+     * Set the offerExpiresAfter property: The number of seconds after which any offers created under this policy
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/DistributionPolicy.java:90]: The duration after

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/PassThroughWorkerSelectorAttachment.java [hunk 3]
```diff
@@ -78,19 +79,19 @@ public LabelOperator getLabelOperator() {
...
-        return this.expiresAfterSeconds;
+    public Duration getExpiresAfter() {
+        return this.expiresAfter;
     }
 
     /**
      * Set the expiresAfterSeconds property: Describes how long the attached label selector is valid in seconds.
      *
-     * @param expiresAfterSeconds the expiresAfterSeconds value to set.
+     * @param expiresAfter the expiresAfterSeconds value to set.
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/PassThroughWorkerSelectorAttachment.java:89]: Comment

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/PassThroughWorkerSelectorAttachment.java [hunk 4]
```diff
@@ -78,19 +79,19 @@ public LabelOperator getLabelOperator() {
      * @return the expiresAfterSeconds value.
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/PassThroughWorkerSelectorAttachment.java:79]: Comment

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterQueueStatistics.java [hunk 5]
```diff
@@ -31,9 +33,8 @@ public final class RouterQueueStatistics {
      * The estimated wait time of this queue rounded up to the nearest minute, grouped
      * by job priority
      */
-    @Generated
     @JsonProperty(value = "estimatedWaitTimeMinutes")
-    private Map<String, Double> estimatedWaitTimeMinutes;
+    private Map<String, Duration> estimatedWaitTimeMinutes;
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterQueueStatistics.java:37]: Change this to Map<int, Duration> and rename to estimatedWaitTimes

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterWorkerSelector.java [hunk 6]
```diff
@@ -127,19 +126,19 @@ public RouterWorkerSelector setValue(Object value) {
...
-        return this.expiresAfterSeconds;
+    public Duration getExpiresAfter() {
+        return this.expiresAfter;
     }
 
     /**
      * Set the expiresAfterSeconds property: Describes how long this label selector is valid in seconds.
      *
-     * @param expiresAfterSeconds the expiresAfterSeconds value to set.
+     * @param expiresAfter the expiresAfterSeconds value to set.
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterWorkerSelector.java:136]:
```suggestion
     * @param expiresAfter the expiresAfter value to set.
```

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterWorkerSelector.java [hunk 7]
```diff
@@ -127,19 +126,19 @@ public RouterWorkerSelector setValue(Object value) {
...
      */
     @Generated
-    public Double getExpiresAfterSeconds() {
-        return this.expiresAfterSeconds;
+    public Duration getExpiresAfter() {
+        return this.expiresAfter;
     }
 
     /**
      * Set the expiresAfterSeconds property: Describes how long this label selector is valid in seconds.
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterWorkerSelector.java:134]:
```suggestion
     * Set the expiresAfterSeconds property: Describes how long this label selector is valid.
```

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterWorkerSelector.java [hunk 8]
```diff
@@ -127,19 +126,19 @@ public RouterWorkerSelector setValue(Object value) {
      * @return the expiresAfterSeconds value.
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterWorkerSelector.java:126]:
```suggestion
     * @return the expiresAfter value.
```

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterWorkerSelector.java [hunk 9]
```diff
@@ -116,7 +115,7 @@ public Object getValue() {
      * @return the RouterWorkerSelector object itself.
      */
     @Generated
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterWorkerSelector.java:117]: Remove?

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterQueueSelector.java [hunk 10]
```diff
@@ -76,7 +76,7 @@ public LabelOperator getLabelOperator() {
      * @return the value value.
      */
     @Generated
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterQueueSelector.java:78]: Remove these annotations?

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterJob.java [hunk 11]
```diff
@@ -389,7 +385,7 @@ public Map<String, RouterJobAssignment> getAssignments() {
      * @return the tags value.
      */
     @Generated
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/RouterJob.java:387]: Remove these annotations?

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/ReclassifyExceptionAction.java [hunk 12]
```diff
@@ -30,7 +30,7 @@ public final class ReclassifyExceptionAction extends ExceptionAction {
      */
     @Generated
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/ReclassifyExceptionAction.java:31]: Remove these annotations?

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/PassThroughWorkerSelectorAttachment.java [hunk 13]
```diff
@@ -78,19 +79,19 @@ public LabelOperator getLabelOperator() {
...
     }
 
     /**
      * Set the expiresAfterSeconds property: Describes how long the attached label selector is valid in seconds.
      *
-     * @param expiresAfterSeconds the expiresAfterSeconds value to set.
+     * @param expiresAfter the expiresAfterSeconds value to set.
      * @return the PassThroughWorkerSelectorAttachment object itself.
      */
     @Generated
```
Comment (williamzhao87) [inline sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/PassThroughWorkerSelectorAttachment.java:92]: Remove these annotations?

File: sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/implementation/models/CancelExceptionActionInternal.java [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+import com.azure.core.annotation.Generated;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+/** An action that marks a job as cancelled. */
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "kind")
+@JsonTypeName("cancel")
+@Fluent
+public final class CancelExceptionActionInternal extends ExceptionActionInternal {
```
[the remark this answers is not in the document]
Comment (cparisineti) [reply sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/implementation/models/CancelExceptionActionInternal.java:16]: ManualReclassifyExceptionAction has workerSelectors. So, I had to make all ExceptionActions internal.

**Review by williamzhao87 — APPROVED** [review]

**Pull request merged**